contribute/pull-request.md

```diff
@@ -0,0 +1,99 @@
+# Create a pull request
+
+We're excited that you're considering making a contribution to the Grafana project! This document guides you through the process of creating a [pull request](https://help.github.com/en/articles/about-pull-requests/).
+
+## Prerequisites
+
+We know you're excited to create your first pull request. Before we get started, here are some resources you should read first:
+
+- Learn how to start [Contributing to Grafana](https://grafana.com/docs/contribute/overview/).
+- Make sure your code follows the relevant [style guides](/contribute/style-guides).
+
+## Your first pull request
+
+If this is your first time contributing to a open-source project on GitHub, make sure you read about [Creating a pull request](https://help.github.com/en/articles/creating-a-pull-request).
+
+To increase the chance of having your pull request accepted, make sure your pull request follows the guidelines:
+
+- Title and description must match the implementation.
+- Pull request must follow the [style guide](#style-guide).
+- Commits within the pull request should follow the [style guide](#style-guide).
+- Pull request should close one related issue.
+- Pull request should contain necessary tests that verify the intended behavior.
+- If your pull request have conflicts, rebase your branch onto the master branch.
+
+If the pull request fixes a bug:
+
+- Pull request description must include `Closes #<issue>` or `Fixes #<issue>`.
+- Pull request should include tests that replicate the fixed bug, to avoid regressions.
+
+### Frontend-specific guidelines
+
+- Pull request doesn't increase the Angular code base.
+- Pull request doesn't contain uses of `any` or `{}` without comments describing why.
+- Pull request doesn't contain large React components that could easily be split into several smaller components.
+- Pull request doesn't contain backend calls directly from components—use actions and Redux instead.
+- Pull request uses [Emotion](https://github.com/grafana/grafana/blob/master/style_guides/styling.md) for styling.
+
+#### Redux
```

Comment:
Looks like this should be level 3.

As a practice, try to avoid headings with a single subheading. There should be at least two subheadings under head heading, or none.